packages/css/src/link/link.scss

```diff
@@ -19,9 +19,9 @@
 }
 
 .amsterdam-link--standalone {
+  display: inline-block;
   font-size: var(--amsterdam-link-standalone-narrow-font-size);
-  line-height: var(--amsterdam-link-standalon-line-height);
-  padding-block: 0.5rem;
+  line-height: var(--amsterdam-link-standalone-line-height);
```

Comment:
Can we configure [this Stylelint rule](https://stylelint.io/user-guide/rules/no-unknown-custom-properties/) to prevent this in the future?

Reply:
I tried it out, it only seems to work for local css vars. Our component css doesn't have a direct import of the design token css vars, so it can't 'know' whether a css var exists.